.github/workflows/deploy.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy
+
+on:
+  push:
+    tags:
+    - '*.*.*'
+
+jobs:
+  deploy:    
+    runs-on: macos-latest
+
+    env:
+      BINTRAY_SNOWPLOW_MAVEN_USER: ${{ secrets.BINTRAY_SNOWPLOW_MAVEN_USER }}
+      BINTRAY_SNOWPLOW_MAVEN_API_KEY: ${{ secrets.BINTRAY_SNOWPLOW_MAVEN_API_KEY }}
+            
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Build
+        run: ./gradlew assembleRelease
+
+      - name: Get version (git tag)
+        id: get_version
+        run: |
+          echo ${GITHUB_REF/refs\/tags\//}
+          echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: Publish
```

Comment:
You can perform all the logic that is in deploy.sh directly in Github Actions if you want to. 
See: https://github.com/snowplow/snowplow-java-tracker/blob/65d3a0cca839ed6dbc111afcaf4bac639fd41c25/.github/workflows/deploy.yml#L43

Totally optional but would remove the `.scripts`.